src/posts/action.md

```diff
@@ -3,56 +3,56 @@ title: Take action
 description: Ways to help out with pausing AGI development.
 ---
 
-The group of people who are aware of AI risks is still small.
+AI will won't get safer unless we act decisively to push for safety. Choose an activity below from your interest / skill areas and help save the future!
 
-You are now one of them.
+## For everyone
 
-**So your actions matter more than you think.**
+### 1. Encourage governments to act
 
-Here are some examples of what you can do.
+- [**Create an email for a politician**](/email-builder) **in the** [**US**](https://democracy.io/): Emails are completely overpowered. Try to convince your government to work towards a pause or enforce other binding AI safety legislation. Here are some [**lobby tips**](/lobby-tips).
+- **Sign petitions**: [International AI Treaty](https://aitreaty.org), [Ban Superintelligence](https://chng.it/Djjfj2Gmpk), [Demand responsible AI](https://www.change.org/p/artificial-intelligence-time-is-running-out-for-responsible-ai-development-91f0a02c-130a-46e1-9e55-70d6b274f4df), or one of the **national petitions**: [UK](https://petition.parliament.uk/petitions/639956), [AUS](https://www.aph.gov.au/e-petitions/petition/EN5163), [NL](https://aipetitie.nl)
 
-## Inform others
+### 2. Inform others
 
-- **Share** about AI risk on social media. This website might be a good start. <!-- Once we have a intro video that should be our go to. If we are far from that we could share some official e-flyers here? https://drive.google.com/drive/u/1/folders/1c6D_i8U95FUpfrl-eR-oRNoHUf3zghOc -->
-- **Create** [articles](/learn#articles), [videos](/learn#videos) or [memes](https://twitter.com/AISafetyMemes).
-- **Talk** to people in your life about this. Answer their questions, and get them to act. Use our [counterarguments](/counterarguments) to help you be more persuasive.
-- [**Tabling**](/tabling) and [**flyering**](/flyering) are great ways to reach many people in a short amount of time.
+- **Share** about AI risk on your social media. Tag @pauseai (Twitter), or @pause_ai (Instagram) in your posts.
```

Comment:
Good idea

Reply:
I think it was good to have an example of what they could share. Maybe we could link this playlist: https://www.youtube.com/watch?v=xBqU1QxCao8&list=PLI46NoubGtIJa0JVCBR-9CayxCOmU0EJt and say "one of these videos would be a good start"